main.go

```diff
@@ -13,7 +13,9 @@ import (
 func createServer()  {
 	router := mux.NewRouter()
 
-	router.HandleFunc("/", routes.HomePage)
+	router.HandleFunc("/", routes.HomePage).Methods("GET")
+
+	router.HandleFunc("/icecream-test", routes.TestMongoDB).Methods("GET")
 
```

Comment:
When starting developing the endpoints, since we are doing REST, maybe adding a prefix `/api/v1` with version to keep track of API changes in the future

Reply:
That makes sense. 
@yukari-n-erb Please note this when you start working on the endpoints today!

Reply:
Thank you for the information, I added a comment about this!
https://github.com/ice-cream-the-fat-cat/ice-cream-backend/issues/6#issuecomment-905291833